debezium-connector-mysql/src/test/java/io/debezium/connector/mysql/MySqlAntlrDdlParserTest.java

```diff
@@ -612,6 +612,25 @@ public void shouldSupportCreateIndexBothAlgoAndLock() {
         assertThat(((MySqlAntlrDdlParser) parser).getParsingExceptionsFromWalker().size()).isEqualTo(0);
     }
 
+    @Test
+    @FixFor("DBZ-4661")
+    public void shouldSupportCreateTableWithEcrytion() {
+        parser.parse(
+                "CREATE TABLE `t_test_encrypted_test1` " +
+                        "(`id` int(11) NOT NULL AUTO_INCREMENT) " +
+                        "ENGINE=InnoDB DEFAULT CHARSET=utf8 ENCRYPTED=YES COMMENT 'MariaDb encrypted table'",
+                tables);
+        parser.parse(
+                "CREATE TABLE `t_test_encrypted_test2` " +
+                        "(`id` int(11) NOT NULL AUTO_INCREMENT) " +
+                        "ENGINE=InnoDB DEFAULT CHARSET=utf8 `encrypted`=yes COMMENT 'MariaDb encrypted table'",
```

Comment:
now sure how to accept "`" character?

Reply:
```suggestion
                        "ENGINE=InnoDB DEFAULT CHARSET=utf8 encrypted=yes COMMENT 'MariaDb encrypted table'",
```
@ismailsimsek `ENCRYPTED` or `encrypted` are keywords and should not be put inside backtick characters. Else it would be treated as a literal string.

Reply:
@ani-sha agree its odd that it has backtick character. unfortunately `SHOW CREATE TABLE` is generating it with backtick. is there anyway to accept it ?